t/ui/13-admin.t

```diff
@@ -30,18 +30,25 @@ eval 'use Test::More::Color';
 eval 'use Test::More::Color "foreground"';
 
 use File::Path qw/make_path remove_tree/;
+use Module::Load::Conditional qw/can_load/;
 
 my $test_case = OpenQA::Test::Case->new;
 $test_case->init_data;
 
 use t::ui::PhantomTest;
 
-my $driver = t::ui::PhantomTest::call_phantom();
-if ($driver) {
+# skip if phantomjs or Selenium::Remote::WDKeys isn't available
+# we set $enterkey here to avoid using an undefined bareword later if
+# WDKeys (and hence the KEYS const) is not available.
+my $driver   = t::ui::PhantomTest::call_phantom();
+my $enterkey = '';
+if ($driver && can_load(modules => {'Selenium::Remote::WDKeys' => undef,})) {
```

Comment:
Ah right, because autoload and explicit import are not evaluated during compilation. However you can access KEYS directly even without importing it.

Try following: remove `$enterkey` and instead of `KEYS->{'enter'}` use `Selenium::Remote::WDKeys->KEYS->{'enter'}`.